src/igvfd/tests/data/inserts/assay_ontology_term.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "term_id": "OBI:0002041",
+        "term_name": "STARR-seq",
+        "status": "released",
+        "uuid": "df207aeb-3484-3730-b64c-d3fdb4506a5e"
```

Comment:
Do you want to have an insert that uses aliases and notes since those are fields that are included?  I think that makes sense for future developers to have an example of every field.

Reply:
I think that makes sense, will add this